pytest_fixtures/component/satellite_auth.py

```diff
@@ -451,3 +452,32 @@ def enroll_ad_and_configure_external_auth(request, ad_data, target_sat):
     target_sat.execute(f'echo {settings.ldap.password} | {token_command}')
     target_sat.execute('chown root.apache /etc/httpd/conf/http.keytab')
     target_sat.execute('chmod 640 /etc/httpd/conf/http.keytab')
+
+    # enable the foreman-ipa-authentication feature
+    target_sat.execute('satellite-installer --foreman-ipa-authentication=true', timeout='20m')
+
+    # add foreman ad_gp_map_service (BZ#2117523)
+    line_number = int(
+        target_sat.execute(
+            "awk -v search='domain/' '$0~search{print NR; exit}' /etc/sssd/sssd.conf"
+        ).stdout
+    )
+    target_sat.execute(
+        f'sed -i "{line_number + 1}i ad_gpo_map_service = +foreman" /etc/sssd/sssd.conf'
```

Comment:
Consider for sanity's sake logging the file contents after modification, that way if/when there's a test failure you can see this update was successful.